### ISSUE TITLE
PoI is a little high

### ISSUE THREAD

**Issue by jabo38** [crackerhead/ourNEM#1]
On the computer graphic where the MacBook is, the PoI is too high.

**Comment by saulgray** [comment]
agreed, might be a bit misleading, We should get a new picture of what the actual client looks like on launch, but we will have to do that closer to launch
